### PR TITLE
fix for opening files in py3

### PR DESCRIPTION
**Pull request by niphlod**
fixes also https://github.com/web2py/web2py/issues/1451

### CODE REVIEW

**Comment by gi0baro** [comment]
@niphlod ready for merge?

**Comment by niphlod** [comment]
Yep

Il 20 set 2016 11:28, "Giovanni Barillari" <notifications@github.com> ha
scritto:

File: pydal/contrib/portalocker.py [hunk 1]
```diff
@@ -55,9 +55,13 @@
      solution
 
 """
-
+import sys
 import logging
-logger = logging.getLogger("web2py")
+
+PY2 = sys.version_info[2] == 2
```
Comment (ilvalle) [inline pydal/contrib/portalocker.py]: It is ```sys.version_info[0] == 2```

**Comment by niphlod** [comment]
@ilvalle : fixed now. Dunno if tests should be added here or on web2py

**Comment by ilvalle** [comment]
portalocker is used on both pydal and web2py. Placing unittest in one or another is the same, perhaps in web2py is better because language files can be used as input for tests.

**Pull request merged**